city_scrapers/spiders/chi_ssa_21.py

```diff
@@ -0,0 +1,161 @@
+# -*- coding: utf-8 -*-
+from city_scrapers.spider import Spider
+from city_scrapers.constants import COMMISSION
+import dateutil.parser
+
+
+class ChiSsa21Spider(Spider):
+
+    name = 'chi_ssa_21'
+    agency_name = 'Lincoln Square Neighborhood Improvement Program'
+    timezone = 'America/Chicago'
+    allowed_domains = ['www.lincolnsquare.org']
+    start_urls = ['http://www.lincolnsquare.org/SSA-no-21-Commission-meetings']
+
+    def parse(self, response):
+        """
+        `parse` should always `yield` a dict that follows the Event Schema
+        <https://city-bureau.github.io/city-scrapers/06_event_schema.html>.
+
+        Change the `_parse_id`, `_parse_name`, etc methods to fit your scraping
+        needs.
+        """
+        for item in response.xpath('//div[@id="content-327081"]/p'):
+
+            data = {
+                '_type': 'event',
+                'name': self._parse_name(item),
```

Comment:
This is also a bit confusing for SSAs, but it seems like it makes the most sense to use "Lincoln Square Neighborhood Improvement Program" here. We mainly want to avoid repeating information from the agency name, and in other cases we've said "SSA Governing Commission", but it looks like they're using that name here.

You can also get rid of the `_parse_name` method and just return the static string here since it doesn't change